app/scripts/directives/overview/mobileClientRow.js

```diff
@@ -52,6 +55,26 @@
         });
         row.updateServicesInfo();
       });
+
+      DataService.watch(buildsVersion, row.context, function(buildsData) {    
+        // Filter builds by mobile client id
+        var filteredBuilds = _.filter(buildsData.by('metadata.name'), function(build) {
+          return _.get(build, 'metadata.labels.mobile-client-id') === _.get(row, 'apiObject.metadata.name');
+        });
+
+        // Group all builds by buildconfig name
+        var groupedBuilds = _.groupBy(filteredBuilds, function(build) {
```

Comment:
I believe we are showing the last five builds so we aren't grouping by buildconfig